### PR TITLE
DHT: new recipe

### PR DESCRIPTION
**Pull request by extrowerk**

### CODE REVIEW

**Comment by Begasus** [comment]
I'll try to see if I can get around to it tomorrow :)

File: net-libs/dht/dht-0.26.recipe [hunk 1]
```diff
@@ -0,0 +1,72 @@
+SUMMARY="BitTorrent DHT library"
+DESCRIPTION="dht implement the variant of the Kademlia Distributed Hash \
+Table (DHT) used in the Bittorrent network (\"mainline\" variant)."
+HOMEPAGE="https://github.com/transmission/dht"
+COPYRIGHT="2009-2010 by Juliusz Chroboczek"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/transmission/dht/archive/dht-0.26.tar.gz"
```
Comment (korli) [inline net-libs/dht/dht-0.26.recipe:8]:
```suggestion
SOURCE_URI="https://github.com/transmission/dht/archive/dht-$portVersion.tar.gz"
```

File: net-libs/dht/dht-0.26.recipe [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+REQUIRES_devel="
+	dht$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
```
Comment (korli) [inline net-libs/dht/dht-0.26.recipe:39]: is cmd:make really needed?

File: net-libs/dht/dht-0.26.recipe [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+INSTALL()
+{
+	mkdir -p $includeDir/dht $libDir
+	cp dht.h $includeDir/dht
+	cp libdht.so $libDir
+
+	mkdir -p $developLibDir/pkgconfig
+	# No need to call prepareInstalledDevelLibs
+	cat > $developLibDir/pkgconfig/live555.pc << EOF
```
Comment (korli) [inline net-libs/dht/dht-0.26.recipe:56]: live555.pc?

Comment (extrowerk) [reply net-libs/dht/dht-0.26.recipe:56]: meh

**Comment by extrowerk** [comment]
@Begasus : could you please test this 4 new recipe (utp, b64, dht and transmission) on x86? Thanks!

**Comment by Begasus** [comment]
Parse errors on gcc2, gcc8 seems ok

**Pull request merged**